data/mods/Magiclysm/recipes/magic_tools.json

```diff
@@ -44,14 +44,27 @@
     "result": "silver_infuser",
     "category": "CC_ENCHANTED",
     "subcategory": "CSC_ENCHANTED_TOOLS",
-    "skill_used": "fabrication",
-    "skills_required": [ "spellcraft", 3 ],
+    "skill_used": "spellcraft",
+    "skills_required": [ "fabrication", 3 ],
     "difficulty": 4,
-    "time": "20 m",
+    "time": "30 m",
     "book_learn": [ [ "alchemy_basic", 3 ], [ "necro_basic", 4 ], [ "techno_basic", 4 ] ],
-    "qualities": [ { "id": "CHISEL", "level": 1 } ],
+    "qualities": [ { "id": "CHISEL", "level": 2 } ],
     "components": [ [ [ "silver_bracelet", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "gold_infuser",
+    "category": "CC_ENCHANTED",
+    "subcategory": "CSC_ENCHANTED_TOOLS",
+    "skill_used": "spellcraft",
+    "skills_required": [ "fabrication", 4 ],
+    "difficulty": 5,
+    "time": "40 m",
+    "book_learn": [ [ "alchemy_basic", 3 ], [ "necro_basic", 4 ], [ "techno_basic", 4 ] ],
+    "qualities": [ { "id": "CHISEL", "level": 3 } ],
+    "components": [ [ [ "gold_bracelet", 1 ] ] ]
```

Comment:
now that we are getting to sufficiently high levels of mana infusing, i want this to take something in addition to the bracelet. maybe some of the crystallized mana dust that was added fairly recently. 40 minutes also seems a little low for a supposedly complex recipe that you only need to craft once.